lib/sarif/bundle_audit_sarif.rb

```diff
@@ -0,0 +1,31 @@
+module Sarif
+  class BundleAuditSarif < BaseSarif
+    BUNDLEAUDIT_URI = 'https://github.com/securego/gosec'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @logs = @scan_report.to_h[:info][:vulnerabilities]
+      @uri = BUNDLEAUDIT_URI
+      @urls = Set.new
+    end
+
+    def parse_issue(issue)
```

Comment:
The `cvss` (Common Vulnerability Scoring System) in the bundle audit output is important, although it is sometimes an empty string.
Please add the `cvss` in the sarif.

Reply:
Also, it is crucial that you add  `patched_versions` and `unaffected_versions` to the sarif.  
If you want, you can just append them to the `fullDescription` for now.

Reply:
I would also add `type` (like UnpatchedGem) to the start of `fullDescription`.

Reply:
I'm not sure what `osvdb` values look like, but you can add it to the end of `fullDescription`.

Reply:
Can you talk to Josh about whether these should be separate attributes?

Reply:
Good call outs, I think adding these to a properties makes sense long term.  Short term of including them in the details works for me